integration/mediation-tests/tests-mediator-2/src/test/java/org/wso2/carbon/esb/mediator/test/script/AddHeadersUsingNashornJsTestCase.java

```diff
@@ -59,7 +61,8 @@ public void testAddHeaderUsingDomElement() throws Exception {
         String payload = "<m:CheckPriceResponse xmlns:m=\"http://services.samples/xsd\">\n"
                 + "<m:stringHead>STRING</m:stringHead>\n" + "    <m:domHead>DOM</m:domHead>\n"
                 + "<m:omHead>OM</m:omHead>\n" + "</m:CheckPriceResponse>";
-        HttpResponse response = doPost(new URL(getApiInvocationURL("addHeadersWithNashornJs")), payload, httpHeaders);
+        HttpResponse response = doPost(new URL(getApiInvocationURL(ADD_HEADER_API)), payload,
+                httpHeaders);
         Assert.assertTrue((response.getData().contains("DOM")), "Response does not contain " + "the keyword \"DOM\". "
```

Comment:
Here also we can remove '+' between "Response does not contain " and "the keyword \"DOM\". "